jwst/cube_build/ifu_cube.py

```diff
@@ -1341,6 +1341,11 @@ def setup_ifucube_wcs(self):
         if self.instrument == 'NIRSPEC':
             final_lambda_min = min(lambda_min)
             final_lambda_max = max(lambda_max)
+            if self.wavemin_user:
+                final_lambda_min = self.wavemin
```

Comment:
There is a wavemax_user and wavemax because of the different ways the MIRI and NIRSpec teams wanted to set the final wavelength range of the IFUCube. 
Rules for MIRI:
If the user has set the min and max wavelength range then the final cube uses these ranges
If the user has not set the wavelength range then the min and max wavelength are read in from the cube pars reference file. 

Rules for NIRSpec:
If the user has set the min and max wavelength range then the final cube uses these ranges
If the user has not set the wavelength range then the min and max wavelength are determined on the fly from the data. 

These differences have lead to having both self.wavemin and self.wavemin_user (sort of evolved over the years)

A ticket that D Law  submitted:  JP-3047 which besides doing some clean up of unused weighting schemes will also clean up how the min and max wavelength are defined and hopefully they will be the same for both teams. 
I suggest we wait for that ticket to get this issue cleaned up.

Reply:
So, bottom line is that the existing changes are good as is?

Reply:
yes - existing changes are good as is